src/routes/transactions/transactions-history.controller.spec.ts

```diff
@@ -298,55 +305,92 @@ describe('Transactions History Controller (Unit)', () => {
     const safeAddress = faker.finance.ethereumAddress();
     const chainResponse = chainBuilder().build();
     const chainId = chainResponse.chainId;
-    const moduleTransaction: any = getJsonResource(
-      'module-transactions/module-transaction-domain.json',
-    );
-    const multisigTransaction: any = getJsonResource(
-      'multisig-transactions/erc20/transfer-source-data.json',
-    ).results[0];
-    const transfer: any = getJsonResource(
-      'incoming-transfers/native-coin/transfer-source-data.json',
-    ).results[0];
-    const incomingTransaction: any = ethereumTransactionToJson(
-      ethereumTransactionBuilder().with('transfers', [transfer]).build(),
-    );
-    moduleTransaction.txType = 'MODULE_TRANSACTION';
-    multisigTransaction.txType = 'MULTISIG_TRANSACTION';
-    incomingTransaction.txType = 'ETHEREUM_TRANSACTION';
-    const tokenResponse = getJsonResource(
-      'multisig-transactions/erc20/token-source-data.json',
-    );
-    const safe = getJsonResource(
-      'incoming-transfers/native-coin/safe-source-data.json',
-    );
-    const transactionHistoryBuilder = {
-      count: 40,
-      next: `${chainResponse.transactionService}/api/v1/safes/${safeAddress}/all-transactions/?executed=false&limit=10&offset=10&queued=true&trusted=true`,
-      previous: null,
-      results: [moduleTransaction, multisigTransaction, incomingTransaction],
-    };
+    const moduleTransaction = moduleTransactionBuilder()
+      .with('executionDate', new Date('2022-12-14T13:19:12Z'))
+      .with('isSuccessful', true)
+      .with('safe', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
+      .with('data', null)
+      .with('operation', 0)
+      .with('value', faker.number.int({ min: 1 }).toString())
+      .with('moduleTransactionId', 'i5a6754140f0432d3b')
+      .build();
+    const multisigTransactionToAddress = faker.finance.ethereumAddress();
+    const multisigTransactionValue = faker.string.numeric();
+    const multisigTransaction = multisigTransactionBuilder()
+      .with('safe', '0xC22de4CeeAFd9436d20A9b18C5970D680D3498c8')
```

Comment:
I think `safeAddress` should be used in `moduleTransactionBuilder` or `multisigTransactionBuilder` here but I noticed the two addresses differ. Is this correct?

Reply:
True, I'll change this even if it's not necessary in terms of the output because of the mocking, it's better for correctness 👍🏻

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)